src/Service.Tests/DatabaseSchema-MsSql.sql

```diff
@@ -460,6 +471,8 @@ SET IDENTITY_INSERT authors_history OFF
 INSERT INTO revenues(id, category, revenue, accessible_role) VALUES (1, 'Book', 5000, 'Anonymous'), (2, 'Comics', 10000, 'Anonymous'),
 (3, 'Journals', 20000, 'Authenticated'), (4, 'Series', 40000, 'Authenticated');
 
+INSERT INTO books_sold(id,book_name,last_sold_on) values(1,'Awesome Book',GETDATE());
```

Comment:
When using `GETDATE()` how do you distinguish whether a default value was used? e.g. the default is `CURRENT_TIMESTAMP`. Wouldn't the values be the same?

Reply:
Yes, correct. However, I don't think we are concerned with distinguishing between the values. I update the default value to constant datetime.